_pages/en_US/seedminer.txt

```diff
@@ -22,11 +22,15 @@ While this test is not strictly necessary to perform the Seedminer exploit, the
 
 There are several different methods that build on the Seedminer exploit, so even if this test fails you can still use another method.
 
-#### DSiWare Management Test (used for unSAFE_MODE)
+1. With your system powered off, hold the following buttons: (Left Shoulder) + (Right Shoulder) + (D-Pad Up) + (A), then press (Power)
+  + Keep holding the buttons until your console boots into Safe Mode
+  + If your console does not boot into safemode or one of these buttons are broken you will need to use a legacy method at the bottom of this page
```

Comment:
Capitalize Safe Mode (the rest of the guide calls it `Safe Mode`, like in `Release the buttons after a few seconds. Your console will boot into Safe Mode`)